src/foundry/foundry.js/applications/formApplication.d.ts

```diff
@@ -61,7 +62,7 @@ declare abstract class FormApplication<
   get isEditable(): boolean;
 
   /**
-   * @param options - (unused) (default: `{}`)
+   * @param options - (default: `{}`)
    * @override
    */
   getData(options?: Application.RenderOptions): D | Promise<D>;
```

Comment:
Mentioning it here because it doesn't really fit anywhere else: `FormApplication.Data` currently still has `object` typed as `Duplicated<O>`, which is incorrect. It should just be `O`.